src/elements/Grid/index.js

```diff
@@ -111,12 +113,35 @@ export const Container = styled.div`
   `)};
 `;
 
-export const Row = styled(Container)`
+const getSubContainerMargin = size => `margin: 0 ${Math.floor((grid[size].gap / -2) * 100) / 100 + grid[size].type};`;
+
+const StyledSubContainer = styled.div`
   display: flex;
   flex-wrap: wrap;
-  justify-content: space-between;
+  ${getSubContainerMargin('tiny')}
+  ${breakpoints.small(css`
+    ${getSubContainerMargin('small')}
+  `)};
+
+  ${breakpoints.medium(css`
+    ${getSubContainerMargin('medium')};
+  `)};
+
+  ${breakpoints.large(css`
+    ${getSubContainerMargin('large')};
+  `)};
+
+  ${breakpoints.wide(css`
+    ${getSubContainerMargin('wide')};
+  `)};
 `;
 
+export const Row = (props, { children }) => (
```

Comment:
Did you not mean the following?
```suggestion
export const Row = ({ children, ...props }) => (
```

Reply:
Done